test/createExamples.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable */
+
+/**
+    Make sure you have the lates build. Run `gulp` to rebuild.
+    Run `node createExamples.js` to create the icons, slashs and files.
+**/
+
+var favicons = require('../dist/index.js'),
```

Comment:
Any particular reason why using `var` here as opposed to `const`?

Reply:
Indeed, it's 2019.